stubs/certifi/METADATA.toml

```diff
@@ -1,2 +1,2 @@
-version = "0.1"
+version = "2020.4.5"
```

Comment:
```
Traceback (most recent call last):
  File "/home/runner/work/typeshed/typeshed/./tests/check_consistent.py", line 197, in <module>
    check_metadata()
  File "/home/runner/work/typeshed/typeshed/./tests/check_consistent.py", line 172, in check_metadata
    assert version.count(".") == 1, msg
AssertionError: Unsupported Python version 2020.4.5
Error: Process completed with exit code 1.
```

This is the format used by the project to represent versions: https://pypi.org/project/certifi/#history

What is the best way to handle this in typeshed?

Reply:
I think the way it works right now is you just use "2020.4" instead.

Reply:
Sounds good. I adjusted that in the latest revision.